modules/ROOT/pages/lb-whitelists.adoc

```diff
@@ -3,11 +3,55 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-If you want to whitelist IP addresses to your load balancers, you must pass those IP addresses in CIDR notation using the xref:anypoint-platform-cli.adoc#cloudhub-load-balancer-whitelist-add[`load-balancer whitelist add`] command.
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies", title="Runtime Fabric"]
 
-The whitelist works for inbound connections at the load-balancer level only, not at the CN certificate level. Ensure that you pass only IP addresses.
+
+If you want to allow IP addresses access to your dedicated load balancer (DLB), you must pass those IP addresses in CIDR notation.
+
+[NOTE]
+The whitelist works for inbound connections at the load-balancer level only, not at the CN certificate level.
+
+You can add the whitelisted IP addresses when you create the DLB or add them to an existing DLB using either Runtime Manager or the command-line interface.
+
+== Add IPs to Whitelist to an Existing Load Balancer
```

Comment:
Does this title the task of adding IPs that you want/need to whitelist to an existing... or is the task adding IPs to the whitelist of an existing...?
Can the wording be clarified a bit, so that it corresponds to the sentence that follows it?